doc/rados/configuration/ceph-conf.rst

```diff
@@ -23,152 +22,165 @@ configuration options. Make sure to understand the consequences before
+  in which the daemon stores data.
 - :confval:`keyring`, :confval:`keyfile`, and/or :confval:`key`, which can be
   used to specify the authentication credential to use to authenticate with the
-  monitor. Note that in most cases the default keyring location is in the data
+  Monitors. Note that in most cases the default keyring location is in the data
   directory specified above.
 
-In most cases, there is no reason to modify the default values of these
-options. However, there is one exception to this: the :confval:`mon_host`
-option that identifies the addresses of the cluster's monitors. But when
+There is usually no reason to modify the default values of these
+options. However, there is one exception: the :confval:`mon_host`
+option that identifies the addresses of the cluster's Monitors. When
 :ref:`DNS is used to identify monitors<mon-dns-lookup>`, a local Ceph
 configuration file can be avoided entirely.
 
 
 Skipping monitor config
 -----------------------
 
-The option ``--no-mon-config`` can be passed in any command in order to skip
-the step that retrieves configuration information from the cluster's monitors.
-Skipping this retrieval step can be useful in cases where configuration is
-managed entirely via configuration files, or when maintenance activity needs to
-be done but the monitor cluster is down.
+The option ``--no-mon-config`` can be passed to any command in order to skip
+the step that retrieves configuration information from the cluster's Monitors.
+Skipping this config option source can be useful in cases where configuration is
+managed entirely via configuration files, or when maintenance is necessary
+but the Monitor quorum is down.
 
 .. _ceph-conf-file:
 
 Configuration sections
 ======================
 
-Each of the configuration options associated with a single process or daemon
-has a single value. However, the values for a configuration option can vary
-across daemon types, and can vary even across different daemons of the same
-type. Ceph options that are stored in the monitor configuration database or in
-local configuration files are grouped into sections |---| so-called "configuration
-sections" |---| to indicate which daemons or clients they apply to.
+Each configuration option associated with a single process or daemon
+has a single value. The value for a configuration option may be
+set for all daemon types or for only daemons of a given type.
+Ceph options that stored in the Monitor configuration database or in
+local configuration files are grouped into *configuration
+sections* in order to indicate to which daemons or clients they apply.
 
 
 These sections include the following:
 
 .. confsec:: global
 
    Settings under ``global`` affect all daemons and clients
-   in a Ceph Storage Cluster.
+   in a Ceph Storage Cluster.  In some cases an option may
+   need to be set in one or more sections, for one or more daemons,
+   that is not obvious from its name. In such cases the description
```

Comment:
s/that is not obvious from its name/and it may not be obvious from the option's name/

The way that this sentence is written now, the relative clause at the end refers back to "daemons", which I think is not its intended referent.